site/src/sphinx/advanced-zookeeper.rst

```diff
@@ -60,19 +60,41 @@ For more information, please refer to the API documentation of the `com.linecorp
 Automatic service registration
 ------------------------------
 
-Use `ZooKeeperUpdatingListener`_ to register your server to a ZooKeeper cluster:
+Use `ZooKeeperUpdatingListenerBuilder`_ to register your server to a ZooKeeper cluster:
 
 .. code-block:: java
 
     import com.linecorp.armeria.server.ServerListener;
-    import com.linecorp.armeria.server.zookeeper.ZooKeeperUpdatingListener;
+    import com.linecorp.armeria.server.zookeeper.ZooKeeperUpdatingListenerBuilder;
 
     // This constructor will use server's default host name, port and weight.
-    // Use the other constructors to override the defaults.
-    ServerListener listener = new ZooKeeperUpdatingListener(
-            /* zkConnectionStr */ "myZooKeeperHost:2181",
-            /* zNode           */ "/myProductionEndpoints",
-            /* sessionTimeout  */ 10000);
+    // Use `nodeValueCodec` method to override the defaults.
+    ServerListener listener =
+            // Zookeeper Connection String and Zookeeper Node to use
+            new ZooKeeperUpdatingListenerBuilder("myZooKeeperHost:2181", "/myProductionEndpoints")
+            // Session Timeout
+            .sessionTimeout(10000)
+            .build();
+    server.addListener(listener);
+    server.start();
+    ...
+
+You can use `CuratorFramework`_ instance instead of Zookeeper Connection String. In this case, you can use the
```

Comment:
- use -> use an existing
- Connection String -> connection string
- In this case, you can use ..... -> Remove this sentence; it doesn't add anything.